Improve YAMB Procedures (#423)
vatpac-technology/sops: docs/aerodromes/classc/Amberley.md

Aircraft will depart the circuit visually and track overhead YAMB to begin tracking for their initial waypoint.

!!! phraseology
    WOLF03 was assiged the BYRON1 departure with their initial clearance.  
    **AMB ADC**: "WOLF03, make left turn, reach `F190` by COWIE, cleared for takeoff"  
    **WOLF03**: "Make left turn, reach `F190` by COWIE, cleared for takeoff, WOLF03"  

| Departure | Initial Constraint | Route |
| --------- | ----------| --------- |
| Northern 3 | `F190` at BINUP | BINUP MURJO BOBED LESKO MOSSI/ADNUK (Transition) |
| Northern 4 | `F200` at ANKED | ANKED JEZZA BOBED LESKO MOSSI/ADNUK (Transition) |
| Central 3 | `F190` at BINUP | BINUP MURJO BOBED VIRGE |
| Central 4 | `F200` at ANKED | ANKED JEZZA BOBED VIRGE |
| Byron 1 | `F190` at COWIE | COWIE KIWEE ZANET LOTMA SEMAJ |
| Byron 2 | `F200` at TUGUN | TUGUN LAGOB ZANET LOTMA SEMAJ |

!!! note
    Since the Coded Departures are not SIDs, AMB ACD needs to write the name of the coded departure in the Global Ops Field of the FDR and amend the flight plan route to include all relevant tracking points.

> !!! note

Reviewer: ```suggestion
!!! abstract "Reference"
```

>     Charts for the Coded Departures may be in YAMB AD2 Supplements 6.2.25.11 available here: [RAAF AIP page](https://ais-af.airforce.gov.au/australian-aip){target=new}

Reviewer: ```suggestion
    Charts for the Coded Departures are available in: [YAMB FIHA AD2 Supp Section 6.2.25](https://ais-af.airforce.gov.au/australian-aip){target=new}
```

### Runway Operations
Runways 15/33 are the primary runways at YAMB. 

## Helicopter Operations

### Taxiway Alpha

Taxiway Alpha has been divided into portions for helicopter operations as follows;

| Location | Name |
| ------ | ----------|
| Between A2 and A4     | Alpha North  |
| Between A2 and RWY 04/22     | Alpha Centre |
| Between A1 and RWY 04/22     | Alpha South |
| TWY Q and A Junction     | Holding Point Quebec |

<figure markdown>
![YAMB Helicopter Taxiway Alpha](img/yamb_heli_taxiwayalpha.png){ width="600" }
  <figcaption>YAMB Helicopter Taxiway Alpha</figcaption>
</figure>

### Helicopter Circuits

To facilitate helicopter circuits, two areas have been established.

> Choppers East: Utilising the threshold RWY 22 as a HLS, remaining within 600 m of threshold RWY 22

Reviewer: ```suggestion
**Choppers East**:  
Utilising the threshold RWY 22 as a HLS, remaining within 600 m of threshold RWY 22
```

and east of a line parallel to RWY 15/33 intersecting threshold RWY 22  

> Choppers West

Reviewer: ```suggestion
**Choppers West**  
```

Utilising the threshold RWY 04 as a HLS, remaining within 600 m of threshold RWY 04
and west of a line parallel to RWY 15/33 intersecting threshold RWY 04.

!!! note

>     The primary area for continous circuit operations is Choppers East. Due to the displaced distance of the threshold of runway 22 from runway 15/33.

Reviewer: ```suggestion
    The primary area for continuous circuit operations is **Choppers East**, due to the displaced distance of the threshold of runway 22 from runway 15/33.
```

!!! phraseology
    CRNG21 plans to enter R637 via the gate MOSSI for Military Training.  
    **AMB ADC**: "CRNG21, cleared YAMB via MOSSI, flight planned route. Runway 15, Northern 3 Departure. Climb `F190`. Squawk 3601, Departure Frequency 126.20"   

Aircraft will depart the circuit visually and track overhead YAMB to begin tracking for their initial waypoint.

!!! phraseology
    WOLF03 was assiged the BYRON1 departure with their initial clearance.  
    **AMB ADC**: "WOLF03, make left turn, reach `F190` by COWIE, cleared for takeoff"  
    **WOLF03**: "Make left turn, reach `F190` by COWIE, cleared for takeoff, WOLF03"  

| Departure | Initial Constraint | Route |
| --------- | ----------| --------- |
| Northern 3 | `F190` at BINUP | BINUP MURJO BOBED LESKO MOSSI/ADNUK (Transition) |
| Northern 4 | `F200` at ANKED | ANKED JEZZA BOBED LESKO MOSSI/ADNUK (Transition) |
| Central 3 | `F190` at BINUP | BINUP MURJO BOBED VIRGE |
| Central 4 | `F200` at ANKED | ANKED JEZZA BOBED VIRGE |
| Byron 1 | `F190` at COWIE | COWIE KIWEE ZANET LOTMA SEMAJ |

> | Byron 2 | `F200` at TUGUN | TUGUN LAGOB ZANET LOTMA SEMAJ |

Reviewer 2: Do you know if these procedures get updated through successive AIRACs? We won't have any indication that they have without manually checking charts (we don't scrape the RAAF charts)

Author: No idea, these name changes were effective from 28th Nov 2024 according to the AD2 SUP.

Reviewer: > Do you know if these procedures get updated through successive AIRACs? We won't have any indication that they have without manually checking charts (we don't scrape the RAAF charts)

I thought about this too. Since we can't refer to them as the "Byron SID" or the "Northern SID" (since they're not SIDs), it's a bit awkward. Maybe we can refer to them as the "Byron Coded Departure"? Otherwise I'm just happy to leave the number in there and roll the dice on future AIRAC updates. What do you reckon?

Author: I'm happy to roll the dice too, i pretty often take a glance at different AD2 SUPs just to see what's changed. Would be more then happy to create another PR if new updates for the coded departures occur and no one has done so already

Reviewer 2: Yeah great, I'm happy with this. Given that we don't have AIP charts for these procedures, most controllers will directly reference the SOPs (rather than the RAAF TERMA) when providing these instructions, so having the procedure number is beneficial.
@llamavert feel free to submit any AIRAC updates as you find them

!!! note
    Since the Coded Departures are not SIDs, AMB ACD needs to write the name of the coded departure in the Global Ops Field of the FDR and amend the flight plan route to include all relevant tracking points.

!!! abstract "Reference"
    Charts for the Coded Departures are available in: [YAMB FIHA AD2 Supp Section 6.2.25](https://ais-af.airforce.gov.au/australian-aip){target=new}

### Runway Operations
Runways 15/33 are the primary runways at YAMB. 

## Helicopter Operations

### Taxiway Alpha

Taxiway Alpha has been divided into portions for helicopter operations as follows;

| Location | Name |
| ------ | ----------|
| Between A2 and A4     | Alpha North  |
| Between A2 and RWY 04/22     | Alpha Centre |
| Between A1 and RWY 04/22     | Alpha South |
| TWY Q and A Junction     | Holding Point Quebec |

<figure markdown>
![YAMB Helicopter Taxiway Alpha](img/yamb_heli_taxiwayalpha.png){ width="600" }
  <figcaption>YAMB Helicopter Taxiway Alpha</figcaption>

> </figure>

Reviewer 2: Are we able to get a phraseology example of how an instruction may be issued to operate in these areas? And is this for arrivals and departures, or operations in ground effect?

Author: Just operations in ground effect, eg: practising winching operations. But i'd imagine that it can also be nominated as a HLS, however about 90% of the time aircraft would be assumed to land in the Taxiway W junction.

Reviewer 2: Yeah a bit of guidance on what these areas are used for would be great.

I'd imagine it would be fairly close to that, likely 'not above 100ft' perhaps. I'd imagine winching ops would sometimes involve out of ground effect hovering, but it would be up to the pilot to request their particular operations. If we can at least get an indication of what the areas are used for and an example, that will give controllers 90% of the answer, they can work the last 10% out themselves.

Author: There isn't really any public knowledge to which areas are used for which type of operations, but i'll most definitely add an example of phraseology for one of the local helicopter callsigns and requesting operations for certain areas.

Reviewer 2: Yeah easy. When you say that 90% of the time aircraft would be presumed to land in the taxiway Q junction, do you mean inbound aircraft or aircraft who have been operating in those heli areas and are now done?

Author: Both, typically the local helicopter CHOP41 will land on the taxiway Q junction after operations in one of the areas, circuits, or a VFR departure off to some random SAR anchor. This is as all their staff and equipment are based in that location.

Reviewer 2: Ok great, perhaps we could say something along the lines of 'helicopters may be processed as per fixed wing ops or from Holding Point Quebec at pilot request'?

Author: Yeah that sounds good

Author: I've added an example phraseology for clearing an aircraft doing operations within the taxiway alpha areas, minus the squawk as in real life CHOP41 isn't verbally issued one (as per AD2 SUP), and additionally I assumed it wouldn't be required considering the helicopter wouldn't be leaving the circuit area and require identification. 

Haven't put anything in yet regarding > 'helicopters may be processed as per fixed wing ops or from Holding Point Quebec at pilot request' feel free to add it where you feel it suits best.

Reviewer 2: No worries, I'll have a quick squizz

Aircraft who are not planned via those points or who are negative RNAV may be assigned a RADAR SID or a visual departure. 

### Coded Departures (Fast Jets)
Visual Departures are commonly requested by high performance military jets and often processed in the form of a coded departure (e.g. BYRON1 departure), which can be found in the AD2 Sups Page for YAMB. These coded departures provide a corridor with altitude constraints to join the Military Training Areas east of Brisbane, avoiding civilian traffic inbound to Brisbane. 

!!! phraseology
    CRNG21 plans to enter R637 via the gate MOSSI for Military Training.  
    **AMB ADC**: "CRNG21, cleared YAMB via MOSSI, flight planned route. Runway 15, Northern 3 Departure. Climb `F190`. Squawk 3601, Departure Frequency 126.20"   

Aircraft will depart the circuit visually and track overhead YAMB to begin tracking for their initial waypoint.

!!! phraseology
    WOLF03 was assiged the BYRON1 departure with their initial clearance.  
    **AMB ADC**: "WOLF03, make left turn, reach `F190` by COWIE, cleared for takeoff"  

>     **WOLF03**: "Make left turn, reach `F190` by COWIE, cleared for takeoff, WOLF03"  

Reviewer 2: @llamavert 
Just doing a final review now. Is this constraint phraseology required? It seems the coded clearance includes the constraint, which would suggest there's no need to reiterate it with a takeoff clearance

![image](https://github.com/user-attachments/assets/f28d3caa-f4b1-449d-b90c-c29b8c274d1d)

Author: The height requirement was always in the phraseology examples before i made the pull request. I do agree with you however in a VATSIM environment it still may be good to reiterate the height requirement for pilots who may be unaware or have just forgot.

Reviewer 2: Yeah ok, no worries. I might remove it for consistency with other positions.

| Location | Name |
| ------ | ----------|
| Between A2 and A4     | Alpha North  |
| Between A2 and RWY 04/22     | Alpha Centre |
| Between A1 and RWY 04/22     | Alpha South |
| TWY Q and A Junction     | Holding Point Quebec |

!!! phraseology 
    **CHOP41**: "Amberley Delivery, CHOP41, for winching ops within Alpha Centre, request clearance"  
    **AMB ACD**: "CHOP41, Amberley Delivery, cleared to operate within Alpha Centre, not above A001, report ops complete"  
    **CHOP41**: "Cleared for winching ops within Alpha Centre, not above A001, report ops complete"  

<figure markdown>
![YAMB Helicopter Taxiway Alpha](img/yamb_heli_taxiwayalpha.png){ width="600" }
  <figcaption>YAMB Helicopter Taxiway Alpha</figcaption>
</figure>

### Helicopter Circuits

To facilitate helicopter circuits, two areas have been established.

**Choppers East**:  
Utilising the threshold RWY 22 as a HLS, remaining within 600 m of threshold RWY 22
and east of a line parallel to RWY 15/33 intersecting threshold RWY 22  

**Choppers West**  
Utilising the threshold RWY 04 as a HLS, remaining within 600 m of threshold RWY 04

> and west of a line parallel to RWY 15/33 intersecting threshold RWY 04.

Reviewer 2: Also what circuit altitude is flown by helicopters?

Author: According to the AD2 SUP and mentioned in the circuit section;

"b) C-27J, light aircraft and helicopters not above `A010`. "

Reviewer 2: Cheers

Civil and AAFC conduct flying operations within the South Western Training Area (SWTA). The SWTA is divided in half by the Rosewood-Mt Walker-Aratula Road, SWTA Alpha (north) and SWTA Bravo (south). Once established in the SWTA, aircraft will remain OCTA. Pilots may request to enter the CTA and climb above `A025` when operationally required.

Departures:

| **Duty Runway** | **Routing** | **Altitude** |
|-----------------|-----------|--------------|
| 15              | MTWK      | `A025`         |
| 33              | CLVT      | `A025`        |

Arrivals:

| **Duty Runway** | **Routing** | **Altitude**         |
|-----------------|-----------|----------------------|
| 15              | CLVT      | `A025` or `A015`        |
| 33              | MTWK      | `A025` or `A015`       |

!!! phraseology 
    **AMB ADC**: "ASTR203, cleared SWTA Alpha via MTWK, maintain 2500, squawk 7301"  
    **ASTR203**: "Cleared SWTA Alpha via MTWK, maintain 2500, squawk 7301, ASTR203"    

## Coordination
### AMB TCU

'Next' coordination is required from AMB ADC to AMB TCU for all aircraft.

!!! example
    <span class="hotline">**AMB ADC** -> **AMB TCU**</span>: "Next, STAL56, runway 33"  

>     <span class="hotline">**AMB TCU** -> **AMB ADC**</span>: "STAL56, Assigned Heading Right 030, unrestricted"  

Reviewer: ```suggestion
    <span class="hotline">**AMB TCU** -> **AMB ADC**</span>: "STAL56, Right heading 030, unrestricted"  
```

>     <span class="hotline">**AMB ADC** -> **AMB TCU**</span>: "Assigned Heading Heading Right 030, STAL56"  

Reviewer: ```suggestion
    <span class="hotline">**AMB ADC** -> **AMB TCU**</span>: "Right heading 030, STAL56"  
```

### Level Assignment
The Standard Assignable Level from  **AMB ADC** to **AMB TCU** is:  
a) The Lower of `F180` or `RFL` for Aircraft assigned via Procedural or RNAV SID.  
b) `F190` for Aircraft assigned a Coded Departure.

## Charts

> !!! note

Reviewer: ```suggestion
!!! abstract "Reference"
```

| 04     | Left |
| 22     | Left |

### Civil Operations
Civil and AAFC conduct flying operations within the South Western Training Area (SWTA). The SWTA is divided in half by the Rosewood-Mt Walker-Aratula Road, SWTA Alpha (north) and SWTA Bravo (south). Once established in the SWTA, aircraft will remain OCTA. Pilots may request to enter the CTA and climb above `A025` when operationally required.

Departures:

| **Duty Runway** | **Routing** | **Altitude** |
|-----------------|-----------|--------------|
| 15              | MTWK      | `A025`         |
| 33              | CLVT      | `A025`        |

Arrivals:

| **Duty Runway** | **Routing** | **Altitude**         |
|-----------------|-----------|----------------------|
| 15              | CLVT      | `A025` or `A015`        |
| 33              | MTWK      | `A025` or `A015`       |

!!! phraseology 
    **AMB ADC**: "ASTR203, cleared SWTA Alpha via MTWK, maintain 2500, squawk 7301"  

>     **ASTR203**: "Cleared SWTA Alpha via MTWK, maintain 2500, squawk 7301, ASTR203"    

Reviewer 2: This might be better on the TCU page, but a section highlighting the training area would be useful for ADC so they know how to clear an aircraft there

Author: So just confirmed, ADC page should include the overview, and phraseology example. While the TCU page includes routing and phraseology examples?

Reviewer 2: Outbound routing absolutely, good to have for ADC so they can issue appropriate clearances. Inbound routing is good for situational awareness but maybe a note to indicate that TCU will take care of it (I'm assuming this is how it works?). The note about aircraft climbing into CTA above A025 is irrelevant to ADC who only owns to A015, but TCU needs to be aware if this is a common procedure. Also need TCU to be aware of routing so they can issue appropriate inbound clearances (again, assuming this is how its managed).

Author: Sounds good, I will add routing into the TCU and move the climb into CTA note to TCU. Plus add the note for arrivals routing being handled by the TCU into ADC.

Author: Added the routing, climb into CTA, aircraft entering / leaving SWTA procedure.

Reviewer 2: Thanks mate
